EIPS/eip-5320.md

```diff
@@ -0,0 +1,155 @@
+---
+eip: 5320
+title: Harberger Tax NFT
+description: Interface for Harberger Tax compatible NFTs.
+author: Green (@greenlucid)
+discussions-to: https://ethereum-magicians.org/t/eip-5320-harberger-taxes-nft/10084
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-07-23
+requires: 20, 721
+---
+
+## Abstract
+
+The following standard defines an API for NFTs subject to Harberger Taxes within smart contracts. This particular flavor of Harberger Taxes works in the following way:
+
+1. Alice owns an Item, and values at an amount X. 
+2. Alice pays taxes according to a tax rate R to a collector, depending on this self-assesed value.
+3. Bob offers Alice an amount Y. Because Y is higher than X, Alice is forced to sell the Item to him.
+4. Bob self-asseses the value at Z, and now pays taxes over it.
+
+This mechanic encourages actors to correctly value an item: overvaluing leads to excessive taxes, while undervaluing lets third parties forcefully purchase at a discount.
+
+The obvious use case of Harberger Taxes is to be used as a coordination mechanism for real world land allocation. Some other examples include:
+
+- Digital land allocation (ex. video games, or social experiments).
+- Advertisement space in websites.
+- Governance mechanisms that use "seats" subject to Harberger.
+- Social media that requires holding a token to interact.
+
+## Motivation
+
+This proposal provides smart contract interoperability to interact with Harberger Taxes.
+
+## Specification
+
+Every contract implementing this EIP must implement the `ERC5320` and `ERC721` interfaces.
```

Comment:
```suggestion
Every contract implementing this EIP must implement the [EIP-5320](./eip-5320.md) and [EIP-721](./eip-721.md) interfaces.
```

Reply:
EIP-5320 is this EIP itself so it's ok to omit

Reply:
Self-references should be avoided, but since this is referencing a solidity interface defined directly in this EIP, this is fine.